kong/plugins/jwt/handler.lua

```diff
@@ -14,7 +14,7 @@ local JwtHandler = BasePlugin:extend()
 JwtHandler.PRIORITY = 1005
 
 --- Retrieve a JWT in a request.
--- Checks for the JWT in URI parameters, then in the `Authorization` header.
+-- Checks for the JWT in URI parameters, then for a JWT token in a cookie names and then the `Authorization` header.
```

Comment:
Please try to keep lines under 80 character long, as per our contribution guidelines :) Thank you!

Reply:
fixed.